src/installer/records.py

```diff
@@ -111,6 +123,22 @@ def __repr__(self):
             self.path, self.hash_, self.size,
         )
 
+    def __eq__(self, other):
```

Comment:
We'd want to add testa for these as well.

Reply:
Good point, will do.